family_foto/logger.py

```diff
@@ -1,19 +1,38 @@
-import logging
+import logging, os
 from logging.config import dictConfig
 
+LOG_FILE_NAME = 'server_werkzeug.log'
+LOG_PATH = 'instance/log'
+
+if not os.path.exists(LOG_PATH):
+    os.mkdir(LOG_PATH)
+
+file = open(f'{LOG_PATH}/{LOG_FILE_NAME}', 'w+')
+file.close()
```

Comment:
???
what are you doing here?
if you want to create a empty file do `touch`.
if you want to check permissions wrap this into a try-except

Reply:
Oh yeah, I was mixing up commands and forgot, I could do this

Reply:
I think there is no `touch` in python, but I shortened it down.